Misc/NEWS.d/next/Library/2019-07-20-01-17-43.bpo-36161.Fzf-f9.rst

```diff
@@ -0,0 +1 @@
+For thread safetyy, use ``ttyname_r(3)`` instead of ``ttyname(3)`` and ``crypt_r(3)`` instead of ``crypt(3)``.
```

Comment:
This is not correct, though, since you're not using `crypt_r` any more. :P

(I should have seen that last time I looked at the message.)

Reply:
In that case will only leave this PR for the `ttyname_r`, and then create another one for just the error checking of `crypt`, and will change the news message accordingly.